api/docs.go

```diff
@@ -254,6 +254,18 @@ const docTemplate = `{
                         "description": "Comma separated list of statuses to optionally filter on",
                         "name": "status",
                         "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Comma separated list of origins to filter on (red_hat,external)",
```

Comment:
```suggestion
                        "description": "Comma separated list of origins to filter on (external,red_hat)",
```
lets be consistent , as e comes before r, perhaps external,red_hat is best